Source/app.py

```diff
@@ -105,7 +108,9 @@ def executeTest():
         saltDataChk = request.form.get('SaltDataset')
         if saltData:
             saltDataText = request.form['SaltDatasetText']
-            saltData(saltDataText)
+            # find out how many rows are needed and provide them to the salter
```

Comment:
This needs to be done properly, by getting the number of threads from the config. I hardcoded 4 for now.